streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -433,6 +485,8 @@
      * @param valueSerde        value serdes for materializing the aggregated table,
      *                          if not specified the default serdes defined in the configs will be used
      * @param name              the name of the resulted {@link KTable}
+     *
+     * @return a {@link KTable} that contains records with unmodified keys and values that represent the latest (rolling) aggregate for each key
```

Comment:
That is correct. Though for "windowed KTable", I think it is simply a list of tables where each table represents the aggregate results for that window. As for the docstring we are only talking about key value types for windowed operations, while talking about semantics for non-windowed operations, which is inconsistent. More specifically, for non-windowed aggregation, if we say:

```returns a {@link KTable} that contains records with unmodified keys and values for type <R> that represent the latest (rolling) aggregate for each key.```

Then for non-windowed aggregation, we should be equivalently informative:

```returns a windowed {@link KTable} which can be treated as a list of {@code KTable}s where each table contains records with unmodified keys and values for type <R> that represent the latest (rolling) aggregate for each key within that window.```